agent/cmd/cuda-vmm-interpose/interpose.c

```
@@ -0,0 +1,1779 @@
+  }
+  result = function != NULL ? function(address, size, descriptors, count) : unavailable();
+  if (result == CUDA_SUCCESS) {
+    memcpy(mapping->access, descriptors, count * sizeof(*descriptors));
+    mapping->access_count = count;
+  }
+  pthread_mutex_unlock(&state_lock);
+  return result;
+}
+
+CUresult CUDAAPI
+cuMemExportToShareableHandle(
```

Cppcheck (2.21.0)

[style] The function 'cuMemExportToShareableHandle' should have static linkage since it is not used outside of its translation unit.

(staticFunction)